memebox-streamdeck/public/common.js

```diff
@@ -1249,9 +1240,3 @@ function WEBSOCKETERROR(evt) {
   return reason;
 }
 
-const SOCKETERRORS = {
```

Comment:
probably leave that to know which are the errors

Reply:
Done!